test/tuned_models.jl

```diff
@@ -77,18 +78,30 @@ end
     @test report.history == history
 end
 
-@testset_accelerated "accel. resampling" accel (exclude=[CPUThreads],) begin
+@testset_accelerated "accel. (CPUThreads)" accel begin
     tm = TunedModel(model=first(r), tuning=Explicit(),
                     range=r, resampling=CV(nfolds=2),
-                    measures=[rms, l1], acceleration_resampling=accel)
+                    measures=[rms, l1], acceleration= CPUThreads(),
+                    acceleration_resampling=accel)
     fitresult, meta_state, report = fit(tm, 0, X, y);
     history, _, state = meta_state;
     results3 = map(event -> last(event).measurement[1], history)
     @test results3 ≈ results
 end
 
-@testset_accelerated("under/over supply of models", accel,
-                     (exclude=[CPUThreads],), begin
+@testset_accelerated "accel. (CPUProcesses)" accel begin
+    best_index = argmin(results)
+    tm = TunedModel(model=first(r), tuning=Explicit(),
+                    range=r, resampling=CV(nfolds=2),
+                    measures=[rms, l1], acceleration=CPUProcesses(),
+                    acceleration_resampling=accel)
+    fitresult, meta_state, report = fit(tm, 0, X, y);
+    history, _, state = meta_state;
+    results4 = map(event -> last(event).measurement[1], history)
+    @test results4 ≈ results
+end
+
```

Comment:
Nice. You've tested all combinations of the two acceleration params and they are giving the same results.